clustered/client/src/test/java/org/ehcache/clustered/client/replication/ActivePassiveClientIdTest.java

```diff
@@ -139,7 +140,8 @@ public void messageTrackedAndRemovedWhenClientLeaves() throws Exception {
 
     storeProxy.getAndAppend(42L, createPayload(42L));
 
-    Map<Long, EhcacheEntityResponse> responses = activeMessageHandler.getTrackedResponsesForSegment(KEY_ENDS_UP_IN_SEGMENT_11, activeMessageHandler.getTrackedClients().findFirst().get());
+    Map<Long, EhcacheEntityResponse> responses = activeMessageHandler.getRecordedMessages().filter(r->r.getClientSourceId().toLong() == activeMessageHandler.getTrackedClients().findFirst().get().toLong())
```

Comment:
Not your code originally but the filtering and mapping seems pretty pointless:
```java
assertThat(activeMessageHandler.getRecordedMessages().collect(toList())).hasSize(1);
```

Reply:
Ditto for all the similar lines in this class.

Reply:
fixed